<launch>

	<node pkg="bitathome_hardware_control" name="hc_cmd_interface" type="hc_cmd_interface.py" />

	<node pkg="joy" name="joy_node" type="joy_node" output="screen" />

	<node pkg="bitathome_move_control" name="hc_joy_control" type="hc_joy_control.py" output="screen" />

</launch>
